app-modules/etno/src/Http/Resources/ItemResource.php

```
@@ -79,6 +79,10 @@ public function toArray(Request $request): array
             'keywords' => KeywordResource::collection($this->whenLoaded('keywords')),
             'research_collections' => ResearchCollectionResource::collection($this->whenLoaded('researchCollections')),
             'document_id' => $this->document_id,
+            'first_media' => $this->whenLoaded('firstMedia', fn () => $this->when(
+                Gate::allows('viewMedia', $this->resource),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Gate' in method 'toArray'. (undefined)

(StaticAccess)